src/integration-test/java/com/commercetools/sync/integration/commons/utils/ProductITUtils.java

```diff
@@ -229,5 +235,38 @@ public static Type createPricesCustomType(
         typeKey, locale, name, ResourceTypeIdsSetBuilder.of().addPrices(), ctpClient);
   }
 
+  /**
+   * Builds a {@link PriceDraft} element
+   *
+   * @param amount the amount to create a price draft.
+   * @param currencyUnits the currency unit of the amount.
+   * @param customerGroupId the customer Group Id to create the reference.
+   * @param validFrom
+   * @param validUntil
+   * @param channelId the channel Id to create the reference.
+   * @param customFieldsDraft the custom fields to set on the resulting price draft.
+   * @return a {@link PriceDraft} element
+   */
+  @Nonnull
+  public static PriceDraft getPriceDraft(
```

Comment:
It would rather name it as `createPriceDraft`

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/734/commits/9cd16dc2132b5b42989bf982ee26b349f1ef557d